community/kernel/src/main/java/org/neo4j/kernel/impl/store/NeoStore.java

```diff
@@ -176,6 +178,21 @@ protected void triggered( Void event )
                         lastCommittedTx, lastClosedTx ) );
             }
         };
+        counts.setInitializer( new DataInitializer<CountsAccessor.Updater>()
+        {
+            @Override
+            public void initialize( CountsAccessor.Updater updater )
+            {
+                stringLogger.warn( "Missing counts store, rebuilding it." );
+                new CountsComputer( NeoStore.this ).initialize( updater );
+            }
+
+            @Override
+            public long initialVersion()
+            {
+                return getLastCommittedTransactionId();
```

Comment:
should we cache the result of `initialVersion` to make it consistent wrt multiple invocations?